### ISSUE TITLE
How to build Linux desktop app?

### ISSUE THREAD

**Issue by Oleg26Dev** [nativefier/nativefier#1461]
### Homework

- [X] I took the time to write a good, descriptive issue title
- [X] I read `nativefier --help` and [API.md](https://github.com/nativefier/nativefier/blob/master/API.md).
- [X] I checked [CATALOG.md](https://github.com/nativefier/nativefier/blob/master/CATALOG.md) for community suggestions & workarounds.
- [X] I searched [existing issues, open & closed](https://github.com/nativefier/nativefier/issues?q=is%3Aissue). Yes, my question is new.
- [X] I'm running the [latest version](https://github.com/nativefier/nativefier/releases).

### Your question

Great!

I have built Blazor site app, then converted the site to a Desktop app - it works fine!

1. I work in windows and have converted the site to the windows desktop app. 
 How to convert it to Linux desktop app? Can I do it in Windows?

2. My test Web app uses Logger and I see log records when I run the app. Is it possible to view Log records in the desktop app?

### Steps to reproduce

>nativefier 'localhost:7042'

### Nativefier version

49.01

### Node.js version

v16.16.0

### npm version

8.19.2

### OS

Windows 10

**Comment by lilyx13** [comment]
Hey, I followed [these instructions on opensource.com](https://opensource.com/article/21/11/linux-apps-nativefier) to install penpot and it worked great (example uses other apps).

One thing I noticed, if you're using gnome you'll see a kinda ugly title in the panel, you can clean that up by tweaking the title in the 'resources/app/package.json' file.

Otherwise it's pretty straightforward. I also set the --width and --height to 1920 and 1080 respectively to start the app in full screen.

I hope this helps!

**Comment by Oleg26Dev** [comment]
Thank you @lilyx13 .
It is interesting and useful link.
But can I build the Linux desktop app on WIndows OS and then provide it to my Linux customer?
Or I should have Linux OS to do it?

**Comment by BastelPichi** [comment]
> Thank you @lilyx13 . It is interesting and useful link. But can I build the Linux desktop app on WIndows OS and then provide it to my Linux customer? Or I should have Linux OS to do it?

You need to have linux, but you can package FOR Windows FROM Linux. You can use VirtualBox, WSL or similar to not to have to install linux on a physical machine.

**Comment by YurkinPark** [comment]
> Hey, I followed [these instructions on opensource.com](https://opensource.com/article/21/11/linux-apps-nativefier) to install penpot and it worked great (example uses other apps).

Yes, description looks good, but i still have something i'd like to improve there: i created .desktop file with all the instructions inside and i see this icon in applications list, but on Gnome i can pin icon to the bottom menu (i'm using this to operate between windows using <Super>+NumKey combination. I still can pin this icon there but it runs subprocess, so, i still can not open application with second calling of this icon

![image](https://user-images.githubusercontent.com/1950562/236020942-d4e3b599-dbaa-4f0d-aab8-a47dd3980222.png)

On the picture: 1 is what i'm clicking (and like to click again to open this application) to start the application, 2 icon of subprocess which exactly starts

Any idea of how to fix this? May be i should change something in .desktop file?

**Comment by YurkinPark** [comment]
It is not me who runs subprocess. Desktop entry does it. So, thats why i have 2 icons after i'm clicking on entry 1. I meant that instructions provided on opensource.com (with desktop entry creation) are not well because i have this situation.
